src/app-nordigen/banks/ing-pl-ingbplpw.js

```diff
@@ -0,0 +1,45 @@
+import { printIban, amountToInteger } from '../utils.js';
+
+/** @type {import('./bank.interface.js').IBank} */
+export default {
+  institutionId: 'ING_PL_INGBPLPW',
+
+  normalizeAccount(account) {
+    return {
+      account_id: account.id,
+      institution: account.institution,
+      mask: account.iban.slice(-4),
+      name: [account.product, printIban(account)].join(' ').trim(),
+      official_name: account.product,
+      type: 'checking'
+    };
+  },
+
+  sortTransactions(transactions = []) {
+    return transactions.sort((a, b) => {
+      return (
+        Number(b.transactionId.substr(2)) - Number(a.transactionId.substr(2))
+      );
+    });
+  },
```

Comment:
`bookingDate` is an [optional](https://nordigen.com/en/docs/account-information/output/transactions/) field and it's not available for all banks. We can where it's available in [this spreadsheet from the Nordigen documentation](https://docs.google.com/spreadsheets/d/1ogpzydzotOltbssrc3IQ8rhBLlIZbQgm5QCiiNJrkyA/edit#gid=489769432).

Reply:
Would it make sense to implement a default `sortTransactions` that is picked up if no bank-specific `sortTransactions` is set?

Asking because I started adding N26 and it has the `bookingDate` field. So it would be a prime candidate for this default case.

Reply:
(maybe just as a `utils.sortByBookingDate` that can be dropped in as `sortTransactions: sortByBookingDate`?)

Reply:
I see most of the banks have the `bookingDate` field. I question is if we have the same format there 🤔 but I hope that yes. A single issue that I can see is that transactions done on a single day will not be sorted correctly because they have the same value in this field. We could use the field `bookingDateTime` but is very rarely available